tools/train_net.py

```diff
@@ -94,16 +106,18 @@ def main():
     start_iter = 0
     max_iter = config.TRAIN.MAX_ITER
     best_param_group_id = get_lr_group_id(optimizer)
+    
+    get_module = lambda model: model.module if distributed else model
```

Comment:
Same for this. Please write it in a function (not using lambda function) and also put it in [utils.py](https://github.com/bowenc0221/panoptic-deeplab/blob/master/segmentation/utils/utils.py).